README.md

```diff
@@ -575,6 +575,7 @@ Apache 2.0; see [`LICENSE`](/LICENSE) for details.
 [gcb-clang-tidy-shield]: https://storage.googleapis.com/cloud-cpp-community-publiclogs/badges/google-cloud-cpp/main/clang-tidy.svg
 [github discussions]: https://github.com/googleapis/google-cloud-cpp/discussions
 [github issues]: https://github.com/googleapis/google-cloud-cpp/issues
+[github-issue]: https://github.com/googleapis/google-cloud-cpp/issues/new/choose
```

Comment:
nit: feels like we should be able to avoid the duplication here.

Reply:
Maybe?  (The distinction being between perusing existing issues and filing a new one.)